charts/kubernetes-agent/values.yaml

```diff
@@ -97,3 +97,10 @@ resources: {}
 # requests:
 #   cpu: 100m
 #   memory: 128Mi
+
+
+# Used for integration testing to avoid registring with a real Octopus Server
+testing:
```

Comment:
Out of curiosity, what value does the test inject?

Reply:
https://github.com/OctopusDeploy/OctopusTentacle/pull/870/files?w=1#diff-e67e3ac4eb073effa4a495a05b2cf8f01b175807e075f3d88728b5353a0f3e21R88-R94